### PR TITLE
Updates regarding RHDEVDOCS-4758

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Allow admins to define pre-pinned resources on Dev navigation
Doc JIRA: https://issues.redhat.com/browse/RHDEVDOCS-4758
Dev JIRA: https://issues.redhat.com/browse/ODC-5012
Fix Version: Openshift 4.13 Freeze

Doc Preview: https://55298--docspreview.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console.html#odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_customizing-web-console

SME Review: @jerolimov 
QE Review: @hemantsaini-7 
Peer Review: @rolfedh

### CODE REVIEW

**Review by christoph-jerolimov — APPROVED** [review]
Lgtm!

**Comment by hemantsaini-7** [comment]
lgtm

File: modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc [hunk 1]
```diff
@@ -12,13 +12,14 @@ You can customize a developer catalog by using the form view in the Web Console.
...
 * An OpenShift web console session with cluster administrator privileges.
+* The Developer perspective is enabled.
 
 .Procedure
 
 . In the *Administrator* perspective, navigate to *Administration* -> *Cluster Settings*.
 . Select the *Configuration* tab and click the *Console (operator.openshift.io)* resource.
 . Click *Actions* -> *Customize*.
-. In the respective sections, enable or disable the items in the list.
+. In the *Pre-pinned navigation items*, *Add page*, and *Developer Catalog* sections, enable or disable the items in the list.
```
Comment (rolfedh) [inline modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc:22]: Maybe 
```suggestion
. Enable or disable items in the *Pre-pinned navigation items*, *Add page*, and *Developer Catalog* sections.
```

File: modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc [hunk 2]
```diff
@@ -27,6 +28,11 @@ image::odc_customizing_developer_catalog.png[]
 
 [NOTE]
 ====
+As an administrator, you can define which navigation items will display by default for all users. You can also reorder the navigation items.
```
Comment (rolfedh) [inline modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc:31]:
```suggestion
As an administrator, you can define the navigation items that appear by default for all users. You can also reorder the navigation items.
```
* IBM Style: "Write in the simple present tense as much as possible."
* IBM Style: "display verb: Use only as a transitive verb. For example, write "the system displays a message" or "the message is displayed" (not "the message displays")."

**Review by rolfedh — APPROVED** [review]
Hi Shreya. Nice work. I've offered a couple of suggestions and related guidelines.

**Comment by rolfedh** [comment]
Pre-approved to expedite.

**Pull request merged**